backend/pkg/backend/backend.go

```diff
@@ -87,6 +88,12 @@ func Run() {
 		go dbHandler.CreateFakeData()
 	}
 
+	dbHandler.SetMaterializedViewHandler(dbConfig)
```

Comment:
Do we want to set it in case of local DB? (sqlite) I dont see it is being used in that case